ctru-rs/src/error.rs

```diff
@@ -48,13 +73,20 @@ impl<T> FromResidual<Error> for Result<T> {
     }
 }
 
-/// The error type returned by all libctru functions.
+/// The generic error enum returned by [`ctru-rs`](crate) functions.
+///
+/// This error enum supports parsing and displaying [`ctru_sys::Result`] codes.
 #[non_exhaustive]
 pub enum Error {
+    /// Raw [`ctru_sys::Result`] codes.
     Os(ctru_sys::Result),
+    /// Generic `libc` error codes.
```

Comment:
Same question here, could this be `` [`libc`] `` ? Also since it's a string probably would just say `errors` instead of `error codes`.